opensrp-opd/src/main/java/org/smartregister/opd/utils/ConstantsUtils.java

```diff
@@ -0,0 +1,252 @@
+package org.smartregister.opd.utils;
+
+/**
+ * Created by Ephraim Kigamba - ekigamba@ona.io on 2019-09-30
+ */
+
+public class ConstantsUtils {
```

Comment:
This is in ANC could you extend it

Reply:
ANC import is going to be used for a single class. It's going to be removed later on